msl_expressions/autogenerated/src/Plans/Behaviours/Intercept.cpp

```diff
@@ -113,12 +113,8 @@ namespace alica
         shared_ptr < geometry::CNPosition > predPos = make_shared < geometry::CNPosition > (0.0, 0.0, 0.0);
         double timestep = 33;
         double rot = od->motion.rotation * timestep / 1000.0;
-        for (int i = 1; i * timestep < 160; i++)
+        for (int i = 1; i < 6; i++)
```

Comment:
The old loop only goes from i = 1 to i = 4, since 4 * 33 = 132 and 5 * 33 = 165 > 160.

So i <= 4 or i < 5 would be right.